src/components/PupilViews/PupilProgrammeListing/PupilProgrammeListing.view.tsx

```diff
@@ -40,101 +47,181 @@ export const PupilViewsProgrammeListing = ({
+        return [yearDescriptions];
+    }
+  };
+
+  const BrowseOptions = () => {
+    switch (true) {
+      // examboard is chosen and there are multiple tiers
+      case chosenExamboard !== null && tiers.length > 1:
+        return (
+          <BrowseTierSelector
+            tiers={tiers as TierData[]}
+            baseSlug={baseSlug}
+            examboardSlug={chosenExamboard?.examboardSlug} // TS complains chosenExamboard could be null ?!
+            isLegacy={isLegacy}
+            phaseSlug={phaseSlug}
+          />
+        );
+      // examboard is not chosen and there are multiple tiers
+      case examboards.length > 1 &&
+        chosenExamboard === null &&
+        tiers.length > 1:
+        return (
+          <BrowseExamboardSelector
+            examboards={examboards}
+            baseSlug={baseSlug}
+            onClick={(examboard) => setChosenExamboard(examboard)}
+            isLegacy={isLegacy}
+            phaseSlug={phaseSlug}
+          />
+        );
+      // examboard is not chosen and there is only one or no tiers
+      case examboards.length > 1 &&
+        chosenExamboard === null &&
+        tiers.length <= 1:
+        return (
+          <BrowseExamboardSelector
+            examboards={examboards}
+            baseSlug={baseSlug}
+            isLegacy={isLegacy}
+            phaseSlug={phaseSlug}
+          />
+        );
+      // there are only tiers
+      case examboards.length <= 1 && tiers.length >= 1:
+        return (
+          <BrowseTierSelector
+            tiers={tiers as TierData[]}
+            baseSlug={baseSlug}
+            isLegacy={isLegacy}
+            phaseSlug={phaseSlug}
+          />
+        );
+      default:
+        return <div>No programme factors to be selected</div>;
+    }
+  };
+
+  const optionTitles = (): { hint: string; title: string } => {
+    switch (true) {
+      case examboards.length > 1 && chosenExamboard === null:
+        return {
+          hint: "An exam board sets and assesses what you learn in your GCSE course. If you can't find your exam board here or are not sure which exam board you are studying, ask your teacher.",
+          title: "Choose an exam board",
+        };
+      default:
+        return {
+          hint: "Some GCSE subjects have a foundation and higher tier.  Different content may be covered in each tier so it is important to check with your teacher if you are unsure which tier you are doing.",
+          title: "Choose a tier",
+        };
+    }
+  };
+
+  const optionTitleSlot = (
+    <OakFlex $alignItems={"center"} $gap={"space-between-xs"}>
+      <OakInfo tooltipPosition="top-left" hint={optionTitles().hint} />
+      <OakHeading $font={"heading-5"} tag="h1">
+        {optionTitles().title}
+      </OakHeading>
+    </OakFlex>
+  );
+
   return (
     <OakThemeProvider theme={oakDefaultTheme}>
-      <OakFlex
-        $flexDirection={"column"}
-        $gap={"space-between-m"}
-        $pa={"inner-padding-m"}
+      <AppLayout
```

Comment:
Could we keep this at the page level. It's confusing when we're inconsistent. Should be cut and pasteable.